layouts/shortcodes/glossary_tooltip.html

```diff
@@ -17,7 +17,7 @@
 
 {{- $tooltip := $tooltip | replaceRE "(?s)<a class='glossary-tooltip'.*?>(.*?).*</a>" "$1" | plainify -}}
 {{- $tooltip := trim $tooltip " \n" -}}
-<a class='glossary-tooltip' title='{{- $tooltip | safeHTML -}}' data-toggle='tooltip' data-placement='top' href='{{ $external_link }}' target='_blank' aria-label='{{ $text }}'>
+<a class='glossary-tooltip' title='{{- $tooltip | safeHTML -}}' data-toggle='tooltip' data-placement='top' href='{{ $external_link | relLangURL }}' target='_blank' aria-label='{{ $text }}'>
```

Comment:
If someone wants to link to an English page explicitly (because there's no localized version of it yet), does that still work?

Reply:
It doesn't matter due to the rules upon which the glossary page is built.

Let's take the Polish language as an example, https://deploy-preview-46306--kubernetes-io-main-staging.netlify.app/pl/docs/reference/glossary/?all=true#term-api-group — there is no file `content/pl/docs/reference/glossary/api-group.md` and English version is incorporated instead.

Reply:
Another example, Chinese version of Dockershim glossary term references Docker Engine. 
<img width="1382" alt="Dockershim" src="https://github.com/kubernetes/website/assets/369696/02ac66d0-466d-49d4-9f3f-28b77db288e2">

https://deploy-preview-46306--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/glossary/?fundamental=true#term-dockershim

Docker Engine has  `full_link: https://docs.docker.com/engine/ ` and the link works as expected 🎸 with the proposed in this PR amendment.

https://github.com/kubernetes/website/blob/54ffdac05ef7d92103bd6e5760a07a7911859b14/content/zh-cn/docs/reference/glossary/docker.md?plain=1#L2-L7